apps/dashboard/src/app/(dashboard)/dashboard/components/sidebar/sidebar.client.tsx

```diff
@@ -36,8 +36,8 @@ export const SidebarNav: React.FC<SideBarNavProps> = ({ sections }) => {
   return (
     <aside className="w-full md:w-[300px] border-b md:border-b-0 relative md:sticky top-0 flex-shrink-0">
       {/* mobile toggle */}
-      <div className="flex flex-row md:hidden w-full justify-between items-center p-4">
-        <label className="flex items-center text-accent-foreground font-semibold text-sm pl-3 border-l-2 border-primary h-8">
+      <div className="flex flex-row items-center justify-between w-full p-4 md:hidden">
```

Comment:
is this just ordering changes?

Reply:
Yeah I had some changes in here but removed them, the formatter probably reordered things in the process